### PR TITLE
[planet-finance] New Voting Strategy

### PR DESCRIPTION
**Pull request by ash-greninja-poke**
Fixes # .

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

**Comment by ash-greninja-poke** [comment]
will you guys merge it to the existing voting strategy?

**Comment by ash-greninja-poke** [comment]
@ChaituVR @bonustrack please merge it.

**Comment by bonustrack** [comment]
> will you guys merge it to the existing voting strategy?
@ash-greninja-poke Hi, was that you that created that strategy previously?

**Comment by ash-greninja-poke** [comment]
> > will you guys merge it to the existing voting strategy?
> > @ash-greninja-poke Hi, was that you that created that strategy previously?

yes i have made some changes in the existing strategy.

File: src/strategies/planet-finance/index.ts [hunk 1]
```diff
@@ -29,38 +29,40 @@ const planetFinanceFarmAbi = [
...
+    {
+      constant: true,
+      inputs: [{ internalType: 'address', name: '', type: 'address' }],
+      name: 'balanceOf',
+      outputs: [{ internalType: 'uint256', name: '', type: 'uint256' }],
+      payable: false,
+      stateMutability: 'view',
+      type: 'function',
+    },
+  ]
```
Comment (ChaituVR) [inline src/strategies/planet-finance/index.ts:54]: If possible replace this with 

```
[
  'function totalSupply() view returns (uint256)',    
  'function balanceOf(address) view returns (uint256)'
]
```

File: src/strategies/planet-finance/index.ts [hunk 2]
```diff
@@ -29,38 +29,40 @@ const planetFinanceFarmAbi = [
     outputs: [{ internalType: 'uint256', name: '', type: 'uint256' }],
     stateMutability: 'view',
     type: 'function'
-  }
+  },
 ];
```
Comment (ChaituVR) [inline src/strategies/planet-finance/index.ts:33]:
```
[
  'function poolInfo(uint256) view returns (address want, uint256 allocPoint, uint256 lastRewardBlock, uint256 accAQUAPerShare, address strat)',
  'function stakedWantTokens(uint256 _pid, address _user) view returns (uint256)'
]
```

File: src/strategies/planet-finance/index.ts (hunk 1 above)
Comment (ash-greninja-poke) [reply src/strategies/planet-finance/index.ts:54]: Not possible

File: src/strategies/planet-finance/index.ts (hunk 2 above)
Comment (ash-greninja-poke) [reply src/strategies/planet-finance/index.ts:33]: Not possible

**Comment by ash-greninja-poke** [comment]
Please merge it

**Comment by ash-greninja-poke** [comment]
What is the error?

**Comment by ash-greninja-poke** [comment]
@bonustrack @ChaituVR 
i guess now it is working please merge it.

**Pull request merged**